internal/collector/event.go

```diff
@@ -106,15 +106,15 @@ func (collector *EventCollector) eventsTotalHandler() cache.ResourceEventHandler
 		AddFunc: func(obj interface{}) {
 			ev := obj.(*v1.Event)
 			// Count only Events created after the exporter starts running.
-			if beforeLatestEvent(startRunning, ev) {
+			if beforeOrEqualLatestEvent(startRunning, ev) {
 				collector.increaseEventsTotal(ev, 1)
 			}
 		},
 		UpdateFunc: func(oldObj, newObj interface{}) {
 			oldEv := oldObj.(*v1.Event)
 			newEv := newObj.(*v1.Event)
 			// Count only Events updated after the exporter starts running.
-			if beforeLatestEvent(startRunning, newEv) {
+			if beforeOrEqualLatestEvent(startRunning, newEv) {
```

Comment:
To stop changing the function name all the time, lets just maybe call this more generic "updatedEvents" maybe? Think this would make sense here:
```
if updatedEvents() {
/// do the things
}
```

WDYT?

Reply:
Indeed, the function name should be improved.
I think "reconciledEvent" would be more appropriate here as we are trying to check if the Event has been reconciled and not updated.